### DISCUSSION TITLE
Misleading about "distance"

### DISCUSSION THREAD

**Discussion by W2Q3Q1** [milvus-io/milvus#17503] [Q&A and General discussion]
As shown in the documentation, there are two distance metrics: L2 & IP. But actually, IP is not a "distance metric", for its result will get higher when two vectors get closer. So, why not using some means to unify their results, such as reciprocal？

**Comment by yhmo** [comment]
Just a habit. There are several metric types use 'distance', such as Euclidean distance, Manhattan distance, Minkowski distance, Hamming Distance. 'distance' is easy to understand when we measuring similarity of embeddings.
